src/renderer/features/governance/components/ReferendumDetails/ProposerName.tsx

```diff
@@ -1,18 +1,17 @@
 import { useStoreMap } from 'effector-react';
 
 import { useI18n } from '@app/providers';
-import { type ChainId, type Referendum } from '@shared/core';
+import { type Referendum } from '@shared/core';
 import { FootnoteText, Shimmering } from '@shared/ui';
 import { referendumService } from '@entities/governance';
 import { AccountAddress } from '@entities/wallet';
 import { detailsAggregate } from '../../aggregates/details';
 
 type Props = {
-  chainId: ChainId;
   referendum: Referendum;
 };
 
-export const ProposerName = ({ chainId, referendum }: Props) => {
+export const ProposerName = ({ referendum }: Props) => {
```

Comment:
Not related to this PR, but I think we should discuss add explorers to the proposer. Now we have address, but we can't go fast to explorer as we have in most of places